src/promptflow/tests/test_configs/flows/assistant-tool-with-connection/add_message_and_run.py

```diff
@@ -60,8 +60,16 @@ async def create_assistant(cli: Union[AsyncOpenAI, AsyncAzureOpenAI], assistant_
 @trace
 async def add_message(cli: Union[AsyncOpenAI, AsyncAzureOpenAI], message: list, thread_id: str):
     content = extract_text_from_message(message)
+    api_args = {
+        "thread_id": thread_id,
+        "role": "user",
+        "content": content
+    }
     file_ids = await extract_file_ids_from_message(cli, message)
-    msg = await cli.beta.threads.messages.create(thread_id=thread_id, role="user", content=content, file_ids=file_ids)
+    if file_ids:
+        api_args['attachments'] = [{"file_id": file_id} for file_id in file_ids]
```

Comment:
What are the other arguments in attachments?

Reply:
![image](https://github.com/microsoft/promptflow/assets/95913588/cbf9b3df-aa8e-42c1-beba-783d86de6159)